pkg/controller/networkpolicy_reconciler.go

```diff
@@ -336,12 +346,33 @@ func (reconciler *NetworkpolicyReconciler) updateDefaultNetworkPolicy(namespace
 }
 
 func (reconciler *NetworkpolicyReconciler) createDefaultNetworkPolicy(namespace string, networkpolicyName string) error {
-	desiredPolicy := reconciler.defaultNetworkPolicies[networkpolicyName].DeepCopy()
-	desiredPolicy.ObjectMeta.Namespace = namespace
+	if _, ok := reconciler.defaultNetworkPolicies[networkpolicyName]; !ok {
+		klog.Infof("Network policy not in buffer. Load %s", networkpolicyName)
```

Comment:
Loading a network policy into memory seems to be something reoccurring. Maybe put this in a separate function e.g. `loadNetworkPolicyFromConfig`.

Reply:
As there will be different logic parts for each policy (like set host network)it will change in the near future.

Reply:
In this case, let's just leave it there